File Import Callback Not Supported

When I run `npm run build` in the demo directory, I ran into this error:

```
> @ build /home/kaijun/Documents/uniswap-oracle/demo
> npx ts-node --project scripts/tsconfig.json scripts/compile-contracts

Error: The following errors/warnings were returned by solc:

PriceEmitter.sol:4:1: ParserError: Source "@keydonix/UniswapOracle.sol" not found: File import callback not supported
import { UniswapOracle } from  "@keydonix/UniswapOracle.sol";
^-----------------------------------------------------------^

PriceEmitter.sol:5:1: ParserError: Source "@keydonix/IUniswapV2Pair.sol" not found: File import callback not supported
import { IUniswapV2Pair } from "@keydonix/IUniswapV2Pair.sol";
^------------------------------------------------------------^


    at compileContracts (/home/kaijun/Documents/uniswap-oracle/demo/scripts/compile-contracts.ts:74:10)
    at main (/home/kaijun/Documents/uniswap-oracle/demo/scripts/compile-contracts.ts:108:42)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ build: `npx ts-node --project scripts/tsconfig.json scripts/compile-contracts`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kaijun/.npm/_logs/2022-02-04T16_35_26_018Z-debug.log
```

Any ideas on the cause behind it? I've ran `npm install`.

--- a/demo/contracts/TestWrappers.sol
+++ b/demo/contracts/TestWrappers.sol
@@ -1,9 +1,9 @@
 pragma solidity 0.6.8;
 pragma experimental ABIEncoderV2;
 
-import { BlockVerifier } from "@Keydonix/BlockVerifier.sol";
-import { MerklePatriciaVerifier } from "@Keydonix/MerklePatriciaVerifier.sol";
-import { Rlp } from "@Keydonix/Rlp.sol";
+import { BlockVerifier } from "@keydonix/BlockVerifier.sol";
+import { MerklePatriciaVerifier } from "@keydonix/MerklePatriciaVerifier.sol";
+import { Rlp } from "@keydonix/Rlp.sol";
 
 contract BlockVerifierWrapper {
 	function extractStateRootAndTimestamp(bytes memory input) public view returns (bytes32 stateRoot, uint256 blockTimestamp, uint256 blockNumber) {

--- a/demo/scripts/compile-contracts.ts
+++ b/demo/scripts/compile-contracts.ts
@@ -6,12 +6,12 @@ import { uniswapCompilerOutput } from './uniswap-compiler-output'
 
 const outputFileNamePrefix = 'price-emitter'
 const sourceFiles = [
-	{ key: '@Keydonix/UniswapOracle.sol', path: 'node_modules/@keydonix/uniswap-oracle-contracts/source/UniswapOracle.sol' },
-	{ key: '@Keydonix/IUniswapV2Pair.sol', path: 'node_modules/@keydonix/uniswap-oracle-contracts/source/IUniswapV2Pair.sol' },
-	{ key: '@Keydonix/BlockVerifier.sol', path: 'node_modules/@keydonix/uniswap-oracle-contracts/source/BlockVerifier.sol' },
-	{ key: '@Keydonix/MerklePatriciaVerifier.sol', path: 'node_modules/@keydonix/uniswap-oracle-contracts/source/MerklePatriciaVerifier.sol' },
-	{ key: '@Keydonix/Rlp.sol', path: 'node_modules/@keydonix/uniswap-oracle-contracts/source/Rlp.sol' },
-	{ key: '@Keydonix/UQ112x112.sol', path: 'node_modules/@keydonix/uniswap-oracle-contracts/source/UQ112x112.sol' },
+	{ key: '@keydonix/UniswapOracle.sol', path: 'node_modules/@keydonix/uniswap-oracle-contracts/source/UniswapOracle.sol' },
+	{ key: '@keydonix/IUniswapV2Pair.sol', path: 'node_modules/@keydonix/uniswap-oracle-contracts/source/IUniswapV2Pair.sol' },
+	{ key: '@keydonix/BlockVerifier.sol', path: 'node_modules/@keydonix/uniswap-oracle-contracts/source/BlockVerifier.sol' },
+	{ key: '@keydonix/MerklePatriciaVerifier.sol', path: 'node_modules/@keydonix/uniswap-oracle-contracts/source/MerklePatriciaVerifier.sol' },
+	{ key: '@keydonix/Rlp.sol', path: 'node_modules/@keydonix/uniswap-oracle-contracts/source/Rlp.sol' },
+	{ key: '@keydonix/UQ112x112.sol', path: 'node_modules/@keydonix/uniswap-oracle-contracts/source/UQ112x112.sol' },
 	{ key: 'PriceEmitter.sol', path: 'contracts/PriceEmitter.sol' },
 	{ key: 'TestWrappers.sol', path: 'contracts/TestWrappers.sol' },
 	{ key: 'TestErc20.sol', path: 'contracts/TestErc20.sol' },

